shared/filters/ecosystems-filter/ecosystems-filter.types.ts

```
@@ -0,0 +1,6 @@
+import { PropsWithChildren } from "react";
```

ESLint

[error] 'PropsWithChildren' is defined but never used.

(unused-imports/no-unused-imports)